.changeset/quiet-games-film.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
```

Comment:
```suggestion
"astro": minor
```

It's not technically needed, since the component is experimental, but I'd rather have this in the minor just to avoid disturbing people by accident